packages/appsync-emulator-serverless/schema.js

```diff
@@ -223,7 +223,7 @@ const generateTypeResolver = (
     return response;
   } catch (err) {
     consola.error(`${info.parentType}.${info.fieldName} failed`);
-    consola.error(err);
+    consola.error(err.errorMessage);
```

Comment:
I don't think we can always account on `errorMessage` being defined.  Perhaps we can do something like:

```
consola.error(err.errorMessage || err.stack || err)
```
?